과제2/index2.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>디자인과제2</title>
+    <link rel="stylesheet" href="style2.css">
+</head>
+<body>
+    <div class='container'>
+        <div class='section menu'>
+            <p>[SECTION MENU]</p>
+            </div>
+        <div class='section header'>
+            <p>[HEADER]</p>
+        </div>
+        <div class='section content'>
+            <p>[CONTENT]</p>
+        </div>
+        <div class='section sign-up'>
+            <p>[SIGN UP]</p>
+        </div>
+        <div class='section feature-1'>
+            [FEATURE 1]
+        </div>
+        <div class='section feature-2'>
+            <p>[FEATURE 2]</p>
+        </div>
+        <div class='section feature-3'>
+            <p>[FEATURE 3]</p>
+        </div>
```

Comment:
```suggestion
        <div class='menu'>
            <p>[SECTION MENU]</p>
            </div>
        <div class='header'>
            <p>[HEADER]</p>
        </div>
        <div class='content'>
            <p>[CONTENT]</p>
        </div>
        <div class='sign-up'>
            <p>[SIGN UP]</p>
        </div>
        <div class='feature-1'>
            [FEATURE 1]
        </div>
        <div class='feature-2'>
            <p>[FEATURE 2]</p>
        </div>
        <div class='feature-3'>
            <p>[FEATURE 3]</p>
        </div>
```
.container 안에 모든 div 요소에 .section 이라는 클래스 명이 있어서 (.container div) 로 선택자를 통일해도 될 것 같습니다